src/Domain/Models/Article/ValueObject/ArticleContent.ts

```diff
@@ -0,0 +1,14 @@
+import { ValueObjectError } from '../../../../http/Errors/ValueObjectError'
+
+export class ArticleContent {
```

Comment:
export class ArticleContent {
  readonly value: string;

  constructor(value: string) {
    if (value.length > 100000) {
      throw new ValueObjectError('content is less than 100000');
    }
    this.value = value;
  }
}


const content = new ArticleContent('xxxx');
content.value;